assets/src/edit-story/app/story/effects/useLoadStory.js

```diff
@@ -25,31 +35,50 @@ function useLoadStory( {
 					status,
 					author,
 					slug,
+					date,
+					modified,
+					excerpt: { raw: excerpt },
```

Comment:
I wonder if excerpt is going to be used for Stories? It wasn't in use before at least, IIRC. Perhaps would be good to confirm? (Unless you did already)

Reply:
@pbakaus Thoughts on excerpt?

Reply:
I think this might be useful to support as the excerpt is often used as description of something. But I don't think it's P0.